model/converter/thrift/zipkin/to_domain.go

```diff
@@ -124,14 +124,22 @@ func (td toDomain) transformSpan(zSpan *zipkincore.Span) *model.Span {
 		SpanID:        model.SpanID(zSpan.ID),
 		OperationName: zSpan.Name,
 		ParentSpanID:  model.SpanID(parentID),
-		Flags:         0,
+		Flags:         td.getFlags(zSpan),
 		StartTime:     uint64(zSpan.GetTimestamp()),
 		Duration:      uint64(zSpan.GetDuration()),
 		Tags:          tags,
 		Logs:          td.getLogs(zSpan.Annotations),
 	}
 }
 
+// getFlags takes a Zipkin Span and deduces the proper flags settings
+func (td toDomain) getFlags(zSpan *zipkincore.Span) uint32 {
+	if zSpan.Debug {
+		return 1
```

Comment:
I would prefer the meaning of bits in the `Flags` field to be the same as in the SpanContext's flags.

https://github.com/uber/jaeger-client-go/blob/master/context.go#L32